guacamole/src/main/frontend/src/app/home/directives/guacRecentConnections.js

```diff
@@ -50,13 +50,24 @@ angular.module('home').directive('guacRecentConnections', [function guacRecentCo
 
             // Required services
             var guacHistory       = $injector.get('guacHistory');
+            var preferenceService = $injector.get('preferenceService');
 
             /**
              * Array of all known and visible recently-used connections.
              *
              * @type RecentConnection[]
              */
             $scope.recentConnections = [];
+            
+            /**
+             * Returns whether or not recent connections should be displayed.
+             * 
+             * @returns {Boolean}
```

Comment:
This should be `{boolean}` (or `{!boolean}` if this will never be `null`).

Reply:
Fixed via rebase.